drivers/gles2/rasterizer_canvas_gles2.cpp

```diff
@@ -1997,6 +1997,10 @@ void RasterizerCanvasGLES2::canvas_end() {
 	RasterizerCanvasBaseGLES2::canvas_end();
 }
 
+void RasterizerCanvasGLES2::set_editor_settings(const VisualServer::EditorSettings &p_settings) {
```

Comment:
Just create a function in VisualServer like set_draw_lines_as_rects() , no need to make it more generic